File: cpp/questions/removeDuplicates.cpp
#include <bits/stdc++.h>

using namespace std;

class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        int ns = nums.size();
        int vp = 0;
        for (const auto& n: nums) {
            if (n != nums[vp]) {
                nums[++vp] = n;
            }
        }
        return vp + 1;
    }
};

File: python/questions/removeDuplicates.py
from typing import List


class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        ln = len(nums)
        vp = 0
        for n in nums:
            if n != nums[vp]:
                vp += 1
                nums[vp] = n
        return vp + 1
